src/compiler/chainrules.jl

```diff
@@ -59,7 +59,7 @@ end
 Convert `x` from the format Zygote uses internally to differentials types ChainRules uses.
 """
 @inline wrap_chainrules_input(x) = x
-@inline wrap_chainrules_input(::Nothing) = ChainRules.Zero()
+@inline wrap_chainrules_input(::Nothing) = ChainRules.Zero() # this line is redundant right?
```

Comment:
I.e. not just now but has always been since a `nothing` would get caught in the `ZBack`?

Reply:
If the input was itself something composite likes tuple where not all the values re nothing `(1.5, nothing, (1.4, nothing))` then it wouldn't be caught by `ZBack`